src/cbor.js

```diff
@@ -72,11 +70,15 @@ exports.unmarshal = (input, opts) => {
       if (val instanceof cbor.Tagged) {
```

Comment:
I don't think that will work very well, we want this exact thing, not something that looks like it